pmd-core/pom.xml

```diff
@@ -61,6 +61,34 @@
                     <suppressionsLocation>pmd-core-checkstyle-suppressions.xml</suppressionsLocation>
                 </configuration>
             </plugin>
+
+            <!-- TODO: Move this to pmd-dist, and have all modules in the classpath so language autocompletion works as expected -->
```

Comment:
For PMD 7 an idea is to move the cli implementation into a separate module `pmd-cli` with its dependencies. This would be a natural place to move that logic as well

Reply:
That's interesting… it would help me with integrating the designer, as I can more neatly add the dependency there… My plan for the designer is to include a `pmd designer -v` to show the designer version BTW.